tests/Unit/Middleware/LegacyMiddlewareTest.php

```diff
@@ -48,13 +48,13 @@ public function testRegister()
 
         $middleware->expects($this->once())
             ->method('loadPage')
-            ->with('user_worklog')
+            ->with('users')
```

Comment:
I honestly just picked the first page that seem to make the test work again. Not sure if the test makes more or less sense with my decision.